docker/utils/s3_upload_email_lambda_function.py

```diff
@@ -0,0 +1,47 @@
+import json
```

Comment:
You should put license information at the top like the other python files.

Reply:
done